org.eclipse.m2e.tests/projects/462944/foo/.mvn/jvm.config

```diff
@@ -0,0 +1,4 @@
+-Xmx2048m
+-Xms1024m
```

Comment:
Can you replace the first two lines with single "-Xmx2048m -Xms1024m"? This is how it's supposed to work on command line, so we may as well test for it in eclipse too.